confluent-kafka.yaml

```diff
@@ -9,8 +9,8 @@ package:
   # with the `version:` field.
   # 2. Created a new variable `mangled-package-version` to append `-ccs` to the
   # version.
-  version: 7.8.0.103
-  epoch: 2
+  version: 7.9.0-18-ccs
```

Comment:
Something may not be correct here. It's not filtering out the -css and has appended this to the version? Also curious why our automation did not bump the git SHA

Reply:
This project version is set to use release-monitoring and in that we do not get the commit SHA.

But it seems the release-monitoring is using GitHub backend to fetch info [link](https://release-monitoring.org/project/371645). Not sure why we are not using GitHub ourselves.

I think we can change the update config to use github and maybe also try to simplify the versioning without using version-transform